parser.py

```diff
@@ -0,0 +1,32 @@
+class Parser:
+
+    weather_records = {}
+    weather_attributes = {}
+
+    def parsing_weather_attributes(self, header_record):
+        for attribute in header_record:
+            self.weather_attributes[attribute] = header_record.index(attribute)
+
+    def parsing_records_into_dictionaries(self, records):
+        self.parsing_weather_attributes(records[0])
+        for record in records[1:]:
+            try:
+                year, month, day = record[0].split("-")
+                year = int(year)
+                month = int(month)
+            except (Exception,):
```

Comment:
We discussed that we have to tackle individual exceptions but still you are raising the general Exception.
Reference: https://github.com/saadalish/Weather-Man/pull/2#discussion_r896525310